src/api/api-utils.ts

```diff
@@ -4,16 +4,14 @@ import {useContextStore} from '@/stores/context'
 
 export default {
   apiBaseUrl: () => import.meta.env.VITE_APP_API_BASE_URL,
-  get: (path: string, redirectOnError?: boolean) => {
-
+  get: (path: string) => {
     return axios.get(`${import.meta.env.VITE_APP_API_BASE_URL}${path}`).then(
-
-        data => data
+      response => response.data
```

Comment:
Axios has changed its API in newer versions; to access the response payload you need to pull out the `data` attribute.